docs/SIL.rst

```diff
@@ -5522,6 +5522,14 @@ The second operand may have either object or address type.  In the
 latter case, the dependency is on the current value stored in the
 address.
 
+The optional ``nonescaping`` attribute indicates that no value derived
+from ``value`` escapes the lifetime of ``base``. Derived values
```

Comment:
`value` and `base` is not defined in the text above. It talks about "first operand" and "second operand"